lib/src/components/NxColorPicker/NxColorPicker.scss

```diff
@@ -21,13 +21,13 @@
 
     &:focus-within {
       .nx-color-picker__outline {
-        stroke: var(--nx-swatch-teal-35);
+        stroke: var(--nx-color-color-picker-border-focus);
       }
     }
 
     &:hover {
       .nx-color-picker__color {
-        stroke: var(--nx-color-interactive-border-hover);
+        stroke: var(--nx-color-text-stark);
```

Comment:
It seems coincidental that this results in the correct values, let's make another variable